setup.py

```diff
@@ -63,7 +63,7 @@ def find_resources(directory, subdirectory):
         "joblib>=0.13",
         "mahotas>=1.4",
         "matplotlib>=2.0.0, !=2.1.0",
-        "mysqlclient==1.4.2",
+        "mysqlclient==1.4.5",
```

Comment:
Only this version seems to be working on Python 3.7+. 1.4.4 installs but can't be imported and 1.4.2 didn't work at all.